electron/legendary_utils/library.ts

```diff
@@ -131,14 +140,21 @@ export async function getLegendaryConfig(file: string) {
           )
           const info = isInstalled
             ? installedGames.filter((game) => game.app_name === app_name)[0]
-            : {}
-
+            : {} 
+          
+          const dlc = () => {
```

Comment:
I guess this function could be simplified if you use every instead of map.
For instance:
`const dlc = () => dlcs.every(dlc => dlc === app_name)`